legacy/tests/application/helpers/AirtimeInstall.php

```diff
@@ -145,38 +145,38 @@ public static function DbConnect($p_exitOnError = true)
 
     /* TODO: This function should be moved to the media-monitor
      * install script. */
-    public static function InstallStorageDirectory()
-    {
-        $CC_CONFIG = Config::getConfig();
-        echo '* Storage directory setup' . PHP_EOL;
-        $ini = parse_ini_file(__DIR__ . '/airtime-install.ini');
-        $stor_dir = $ini['storage_dir'];
-        $dirs = [$stor_dir, $stor_dir . '/organize'];
-        foreach ($dirs as $dir) {
-            if (!file_exists($dir)) {
-                if (mkdir($dir, 02775, true)) {
-                    $rp = realpath($dir);
-                    echo "* Directory {$rp} created" . PHP_EOL;
-                } else {
-                    echo "* Failed creating {$dir}" . PHP_EOL;
-
-                    exit(1);
-                }
-            } elseif (is_writable($dir)) {
-                $rp = realpath($dir);
-                echo "* Skipping directory already exists: {$rp}" . PHP_EOL;
-            } else {
-                $rp = realpath($dir);
-                echo "* Error: Directory already exists, but is not writable: {$rp}" . PHP_EOL;
-
-                exit(1);
-            }
-            echo "* Giving Apache permission to access {$rp}" . PHP_EOL;
-            $success = chown($rp, $CC_CONFIG['webServerUser']);
-            $success = chgrp($rp, $CC_CONFIG['webServerUser']);
-            $success = chmod($rp, 0775);
-        }
-    }
+    // public static function InstallStorageDirectory()
```

Comment:
If you're commenting out this functionality, remove it. We have access to it from git anyway